code/modules/jobs/job_types/roguetown/adventurer/types/wretch/wretchedtoiler.dm

```diff
@@ -0,0 +1,128 @@
+// Hedge Mage, a pure mage adventurer sidegrade to Necromancer without the Necromancer free spells and forced patron. More spellpoints, otherwise mostly the same.
+/datum/advclass/wretch/wretchedtoiler
+	name = "Wretched Toiler"
+	tutorial = "The wretched engine of evil churns ever onward - the gears pushed by wretched toilers such as yourself. And toil you shall - until the machinations of thine masters come to fruition."
+	allowed_sexes = list(MALE, FEMALE)
+	allowed_races = RACES_ALL_KINDS
+	outfit = /datum/outfit/job/roguetown/wretch/wretchedtoiler
+	category_tags = list(CTAG_WRETCH)
+	traits_applied = list(TRAIT_RITUALIST, TRAIT_ARCYNE_T2)
+	cmode_music = 'sound/music/combat_bandit_mage.ogg'
+	extra_context = "Choose between 2 options: being an EVIL mastermind or a WRETCHED servant" //choose between master and servant
+
+	// balance isn't real i picked these bc they're funny
+	subclass_stats = list(
+		STATKEY_STR = -1, //YOU ARE WRETCHED!!!
+		STATKEY_CON = -2, //AND YOU WILL +TOIL+!!!!!!!!!!!
```

Comment:
i kinda like the idea of being a wretched supergoon who smiths for the other wretches but smithing stamina loss is super affected by con
maybe the dedicated toiler could flatten their con and maybe get an extra point of endurance at the cost of losing some int to better create the identity of "big strong loyal goon" rather than mastermind with all the brains

Reply:
you could also just give them the malum smithing trait. I'd say thats dumb but you know druid wildforms get graggerite blessings and so on so whatever.

For context that trait just makes it so hammering a anvil uses effectively 0 stamina.

it's TRAIT_FORGEBLESSED